.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,123 @@
+name: automerge auto-cherry-picked pr's
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
+      - edited
+      - ready_for_review
+    branches-ignore:
+      - master
+  pull_request_review:
+    types:
+      - submitted
+    branches-ignore:
+      - master
+  check_suite:
+    types:
+      - completed
+    branches-ignore:
+      - master
+
+
+jobs:
+  automerge:
+    name: Automerge auto-cherry-picked pr
```

Comment:
In Nailgun `Merging can be performed automatically with 2 approving reviews.`, are we planning to disable this first for auto-merge?

Reply:
@jyejare we can remove mentioned checks from nailgun for z-stream cherry picks.

Reply:
@omkarkhatavkar @Gauravtalreja1 2 Approvals rule is not applicable for zStream branches. Its only for master.